src/wp-includes/comment.php

```diff
@@ -3345,7 +3357,11 @@ function _prime_comment_caches( $comment_ids, $update_meta_cache = true ) {
 	if ( ! empty( $non_cached_ids ) ) {
 		$fresh_comments = $wpdb->get_results( sprintf( "SELECT $wpdb->comments.* FROM $wpdb->comments WHERE comment_ID IN (%s)", implode( ',', array_map( 'intval', $non_cached_ids ) ) ) );
 
-		update_comment_cache( $fresh_comments, $update_meta_cache );
+		update_comment_cache( $fresh_comments, false );
+	}
+
+	if ( $update_meta_cache ) {
+		wp_lazyload_comment_meta( $comment_ids );
```

Comment:
Is there any reason that similar can't be done in `update_comment_cache()` too?

I'm in two minds as to whether that's a good idea or a terrible idea so would like your thoughts.

Reply:
If you look at this - https://github.com/WordPress/wordpress-develop/commit/2c6bf77a9e7532c6ac3a03502b44d108cae86179

We could do that here. If you are a developer that is `update_comment_cache`, it is likely you know what you are doing,I don't want to break that for anyone.

TLDR. Let's leave for now and maybe think about this in the future.

Reply:
Fair point, I agree for backward compatibility it's better to not modify that in `update_comment_cache()`, although I wish we could for a cleaner API :)

Reply:
> We could do that here. If you are a developer that is `update_comment_cache`, it is likely you know what you are doing,I don't want to break that for anyone.

Fair enough, thanks.